### DISCUSSION TITLE
OBS 28.0.0-beta2 Advanced Scene Switcher - Missing Tabs??

### DISCUSSION THREAD

**Discussion by David121245712** [WarmUpTill/SceneSwitcher#501] [General]
![Capture](https://user-images.githubusercontent.com/111291523/184661399-25bd31e9-abec-4d09-99e6-79033660ba11.PNG)

I am not sure if this is because of the new version of OBS but I am missing a bunch of tabs according to other videos I have seen on the advanced screen switcher. Is this correct?

**Comment by nacalai** [comment]
Uncheck "hide tabs". As it says, the tabs can be accomplished with macros
anyway

man. 15. aug. 2022, 17:08 skrev David121245712 ***@***.***>:
